### PR TITLE
*: use c-style comments instead of ccp style comments

### PR DESCRIPTION
**Pull request by aabadie**
### Contribution description

This PR moves cpp style comments found in the code base to C-style comments.

### Issues/PRs references

None

### CODE REVIEW

File: drivers/sdcard_spi/sdcard_spi.c [hunk 1]
```diff
@@ -251,9 +251,9 @@ static sd_init_fsm_state_t _init_sd_fsm_step(sdcard_spi_t *card, sd_init_fsm_sta
                     if ((ocr & SYSTEM_VOLTAGE) != 0) {
                         DEBUG("OCR: SYS VOLTAGE SUPPORTED\n");
 
-                        if ((ocr & OCR_POWER_UP_STATUS) != 0) { //if power up outine is finished
+                        if ((ocr & OCR_POWER_UP_STATUS) != 0) { /* if power up outine is finished */
```
Comment (bergzand) [inline drivers/sdcard_spi/sdcard_spi.c]: Maybe move this comment to the line above to keep the line below 80 chars

File: drivers/sdcard_spi/sdcard_spi.c [hunk 2]
```diff
@@ -251,9 +251,9 @@ static sd_init_fsm_state_t _init_sd_fsm_step(sdcard_spi_t *card, sd_init_fsm_sta
...
                             DEBUG("OCR: POWER UP ROUTINE FINISHED\n");
-                            if ((ocr & OCR_CCS) != 0) {         //if sd card is sdhc
+                            if ((ocr & OCR_CCS) != 0) {         /* if sd card is sdhc */
```
Comment (bergzand) [inline drivers/sdcard_spi/sdcard_spi.c]: Same here

File: drivers/sdcard_spi/sdcard_spi.c [hunk 3]
```diff
@@ -801,7 +801,7 @@ static inline int _write_blocks(sdcard_spi_t *card, char cmd_idx, int bladdr, co
                               SD_DATA_TOKEN_CMD_25_STOP);
             DEBUG("_write_blocks: write multi (%d) blocks: [OK]\n", nbl);
 
-            _send_dummy_byte(card); //sd card needs dummy byte before we can wait for not-busy state
+            _send_dummy_byte(card); /* sd card needs dummy byte before we can wait for not-busy state */
```
Comment (bergzand) [inline drivers/sdcard_spi/sdcard_spi.c]: And here

File: sys/crypto/aes.c [hunk 4]
```diff
@@ -1198,7 +1198,7 @@ int aes_encrypt(const cipher_context_t *context, const uint8_t *plainBlock,
 int aes_decrypt(const cipher_context_t *context, const uint8_t *cipherBlock,
                 uint8_t *plainBlock)
 {
-    //setup AES_KEY
+    /*setup AES_KEY */
```
Comment (bergzand) [inline sys/crypto/aes.c]: Missing space after `/*`

File: cpu/lpc2387/lpc23xx-iap.c [hunk 5]
```diff
@@ -32,9 +32,9 @@
 /* pointer to reserved flash rom section for configuration data */
 __attribute((aligned(256))) char configmem[256] __attribute__((section(".configmem")));
 
-static unsigned int iap_command[5];         // contains parameters for IAP command
-static unsigned int iap_result[2];          // contains results
-typedef void (*IAP)(unsigned int[], unsigned int[]);    // typedefinition for IAP entry function
+static unsigned int iap_command[5];         /* contains parameters for IAP command */
+static unsigned int iap_result[2];          /* contains results */
+typedef void (*IAP)(unsigned int[], unsigned int[]);    /* typedefinition for IAP entry function */
```
Comment (bergzand) [inline cpu/lpc2387/lpc23xx-iap.c]: These comments make the line longer than 80 chars, but I'm not sure if moving them a line up makes it more clear.

**Comment by aabadie** [comment]
Comments addressed, @bergzand. Directly rebased and squashed.

File: boards/common/msba2/board_common_init.c [hunk 6]
```diff
@@ -57,16 +57,16 @@ void init_clks2(void)
 
 void watchdog_init(void)
 {
-    WDCLKSEL = 0;                                   // clock source: RC oscillator
-    WDMOD &= ~WDTOF;                                // clear time-out flag
+    WDCLKSEL = 0;                                   /* clock source: RC oscillator */
```
Comment (bergzand) [inline boards/common/msba2/board_common_init.c]: This one is pushed beyond 80 chars, maybe make the indentation match with line 69 (together with the comment on line 61.

File: cpu/lpc2387/lpc23xx-iap.c [hunk 7]
```diff
@@ -148,13 +151,13 @@ uint8_t flashrom_erase(uint8_t *addr)
...
-    iap_command[3] = p3;        // set 3rd param
-    iap_command[4] = p4;        // set 4th param
+    iap_command[0] = code;      /* set command code */
+    iap_command[1] = p1;        /* set 1st param */
+    iap_command[2] = p2;        /* set 2nd param */
+    iap_command[3] = p3;        /* set 3rd param */
+    iap_command[4] = p4;        /* set 4th param */
 
-    ((void (*)())0x7ffffff1)(iap_command, iap_result);      // IAP entry point
+    ((void (*)())0x7ffffff1)(iap_command, iap_result);      /* IAP entry point */
```
Comment (bergzand) [inline cpu/lpc2387/lpc23xx-iap.c]: Pushed beyond 80 chars by the closing tag.

**Comment by aabadie** [comment]
> Two small nitpicks I missed in the initial review

Fixed

**Pull request merged**